app-misc/dateutils/dateutils-0.4.6.ebuild

```diff
@@ -4,12 +4,20 @@
 EAPI=7
 
 DESCRIPTION="command line date and time utilities"
-HOMEPAGE="https://www.fresse.org/dateutils/"
-SRC_URI="https://bitbucket.org/hroptatyr/${PN}/downloads/${P}.tar.xz"
+HOMEPAGE="https://www.fresse.org/dateutils/ https://github.com/hroptatyr/dateutils"
+
+case "${PV}" in
+	9999)
+		inherit git-r3 autotools
```

Comment:
sort

Reply:
Do you mean `inherit autotools git-r3`? Would that make a difference, or is it common to sort eclasses alphabetically?

Reply:
Just style. It'll only make a difference if two eclasses export the same phases or functions, in which case the last one 'wins' because of the order they're sourced in.

Reply:
Great, thanks!

Reply:
(btw no such collision exists here)